livespark-form-modeler/livespark-form-modeler-dynamic-renderer/livespark-form-modeler-dynamic-renderer-client/src/main/java/org/livespark/formmodeler/renderer/client/rendering/renderers/bs3/base/RadioGroupBase.java

```diff
@@ -92,6 +92,11 @@ public void addValidator(final Validator<T> validator) {
         validatorMixin.addValidator(validator);
     }
 
+    @Override
+    public boolean removeValidator(final Validator<T> validator) {
```

Comment:
@cristianonicolai Since the new gwt-bootstrap3 version already has the StringRadioGroup widget, there'so no need to maintain this code. I added this code to have this widget on Forms while using the old Gwt-bootstrap version.

BTW, thank you for upgrading it, I wanted to do that on future commits ;)